tests/dashbio_demos/app_needle_plot.py

```diff
@@ -103,108 +103,258 @@ def description():
 
 def header_colors():
     return {
-        'bg_color': '#009900',
-        'font_color': '#DFE8F3',
-        'light_logo': False
+        'bg_color': '#0D1A51',
+        'font_color': '#FFF',
+        'light_logo': True
     }
 
 
 def layout():
-    return html.Div(
-        className='row',
-        children=[
-            dcc.Store(id='needle-store'),
-            html.Div(
-                className='four columns needle-tabs',
-                children=dcc.Tabs(
-                    id='tabs',
-                    value='tab-data',
-                    children=[
-                        dcc.Tab(
-                            label='Sample Data',
-                            className='needle-tab',
-                            value='tab-data',
+    return html.Div(id='needleplot-body', children=[
+        html.Div(
+            id='needleplot-wrapper',
+            children=dash_bio.NeedlePlot(
+                id='needle-plot',
+                rangeSlider=True
+            )
+        ),
+
+        html.Div(id='needleplot-control-tabs', children=[
+            dcc.Tabs(id='needleplot-tabs', children=[
+                dcc.Tab(
+                    label='About',
+                    value='what-is',
+                    children=html.Div(className='needleplot-tab', children=[
+                        html.H4('What is Needle Plot?'),
+                        html.P('Needle Plot allows you to display mutations in '
+                               'a genome. Due to its similarity to both a barplot '
+                               'and a scatter plot, it can be used to plot '
+                               'datasets that have too many categories for a '
```

Comment:
Asking out of ignorance: is 'category' the bioinformatics term? Does 'everyone' know what it refers to, in the context of genomic data?

Reply:
Needle plots in general follow this definition (http://support.sas.com/documentation/cdl/en/bidbrdug/67143/HTML/default/viewer.htm#n078fn4uh7lvdtn1p9c0hrindwfl.htm). We could add in a more bioinformatics-specific word. However, I think that datasets (e.g., number of mutations for different locations in the gene) that would be used with the needle plot will be this way by nature.

Reply:
I'm only talking about the term 'category' (as present in the data, not the plot). Isn't it... "too many mutations?"

Reply:
I would reword along the lines of "Combining barplot characteristics with scatter plot characteristics, needle plots are best suited to datasets which have too many mutations(?) for a barplot to be meaningful."